examples/getting_started/3_Deep_dive_into_the_anatomy_of_quantum_circuits.ipynb

```diff
@@ -2,35 +2,37 @@
  "cells": [
```

Comment:
<p>Can I ask what is the motivation to change ZZ to CNOT. Could you correct the line number in the link to the BDK repo along with the other changes? Thanks!</p><p> </p><p>It is a bit concerning that there is no mechanism making sure that changes in the BDK repo is reflected here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
I corrected the link. The reason for changing ZZ to CNOT is because ionQ and OQC do not support it. Although commented out, we include the ionQ and OQC device in it. Any user wanting to test that will see error. (I think this is a long standing error. It was not caught because the notebook seemed only tested for SV1)

Reply:
That's a good call! Really appreciate it!